appveyor.yml

```diff
@@ -6,16 +6,13 @@ install:
   # Get the latest stable version of Node.js or io.js
   - ps: Install-Product node $env:nodejs_version
   - ps: $env:BuildNumber= $env:APPVEYOR_BUILD_NUMBER
+  - ps: $env:Configuration="Release"
 init:
   - git config --global core.autocrlf true
 build_script:
-  - dotnet restore
```

Comment:
I am fine doing something complicated for Travis, I don't have much experience, but I don't see why we should replicate this on appveyor if the standard and easy way works fine. Same for packaging and deploying on nuget as long as it's done by appveyor.